docs/ossm/ossm2-vs-ossm3.md

```diff
@@ -0,0 +1,103 @@
+
+This simplification greatly reduces the footprint and complexity of OpenShift Service Mesh, while providing better, production-grade support for observability through Red Hat OpenShift Observability.
+
+## The `Istio` resource replaces the `ServiceMeshControlPlane` resource
+
+While OpenShift Service Mesh 2 used a resource called `ServiceMeshControlPlane` to configure Istio, OpenShift Service Mesh 3 uses a resource called `Istio`. 
+
+The `Istio` resource contains a `spec.values` field that derives its schema from Istio’s Helm chart values. While this is a different configuration schema than `ServiceMeshControlPlane` uses, the fact that it is derived from Istio’s configuration means that configuration examples from the community Istio documentation can often be applied directly to Red Hat OpenShift Service Mesh’s `Istio` resource. The `spec.values` field in the `IstioOperator` resource (which is not part of OpenShift Service Mesh) has a similar format. The `Istio` resource provides an additional validation schema enabling the ability to explore the resource using the OpenShift CLI command `oc explain istios.spec.values`.
+
+## New resource: `IstioCNI`
+
+The Istio CNI node agent is used to configure traffic redirection for pods in the mesh. It runs as a DaemonSet, on every node, with elevated privileges. The Istio CNI agent has a lifecycle that is independent of Istio control planes, and must be upgraded separately. 
+
+In OpenShift Service Mesh 2, the operator deployed an Istio CNI instance for each minor version of Istio present in the cluster and pods were automatically annotated during sidecar injection, such that they picked up the correct Istio CNI. This was enabled by using the Multus CNI plugin and meant that the management of Istio CNI was mostly hidden from users.
+
+OpenShift Service Mesh 3 no longer uses the Multus CNI plugin, and instead runs Istio CNI as a chained CNI plugin. While this simplification provides greater flexibility for network integrations, without Multus, it means that only one instance of Istio CNI may be present in the cluster at any given time and users must manage its lifecycle independent of Istio control planes. 
```

Comment:
we still use Multus, and the usage of Multus is also unrelated to us running only one instance (that part is true). The actual reason we only run one instance in 3.0 is because upstream did not accept our patch that would have allowed running multiple copies of CNI, with the reason given that it's not required as they have different lifecycles (which then turned into us creating a separate CRD for it)